scripts/variantstore/beta_docs/run-your-own-samples.md

```diff
@@ -10,20 +10,41 @@ The [GVS workflow](https://github.com/broadinstitute/gatk/blob/ah_var_store/scri
 
 To learn more about the GVS workflow, see the [Genomic Variant Store workflow overview](./gvs-overview.md).
 
-### What does it require as input?
+## What does it require as input?
 
 - Reblocked single sample GVCF files with specific annotations described below
-- GVCF index files 
+- GVCF index files
 
-While the GVS has been tested with 250,000 single sample whole genome GVCF files as input, only datasets of up to 25,000 whole genomes are being used for beta testing.
+While the GVS has been tested with 410,000 single sample whole genome GVCF files and 190,000 single sample whole exome GVCF files as input, only datasets of up to 25,000 whole genomes and 100,000 whole exomes are currently supported by the beta workflow.
 
-#### Reblocked GVCF files
+### Supported data types
+_* indicates regularly tested and validated_
 
-If your GVCF files have not been reblocked, you can reblock them using the [WARP reblocking workflow](https://github.com/broadinstitute/warp/blob/master/pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl), which is configured in the [ReblockGVCF Terra workspace](https://app.terra.bio/#workspaces/warp-pipelines/ReblockGVCF). 
+**Species**
+- Human data only (hg38)*
+
+**Sequencing details**
+- Illumina short read sequencing*
+- Whole Genome Sequencing*
+- Whole Exome Sequencing*
+- Blended Genome Exome Sequencing*
+
+**Secondary Analysis Methods**
+- DRAGEN 3.7.8*
+- GATK4*
+- DRAGEN-GATK
+- WARP Whole Genome + GATK 3.5
```

Comment:
Is there a WARP Version?

Reply:
We don't regularly test with new WARP data, so this is something that has run through before without issue but is not regularly validated (as indicated by the absence of *). I can look into what versions of WARP we are sure we have run on before.

Reply:
I've been working on trying to find out what WARP version we have data from in our workspace, and have not gotten to the bottom of it yet. However, if I do, I will open a new PR to update this.